### PR TITLE
Handle No Change Set Waiter Error

### PR DESCRIPTION
**Pull request by kolanos**

### CODE REVIEW

File: newrelic_lambda_cli/integrations.py [hunk 1]
```diff
@@ -127,30 +127,28 @@ def create_parameters(
     capabilities = ["CAPABILITY_IAM"]
     if role_name is not None:
         parameters.append({"ParameterKey": "FunctionRole", "ParameterValue": role_name})
-        capabilities = []
     elif mode != "CREATE":
         parameters.append({"ParameterKey": "FunctionRole", "UsePreviousValue": True})
-        capabilities = []
```
Comment (MattWhelan) [inline newrelic_lambda_cli/integrations.py:133]: Why did you remove the capability reduction here?

File: newrelic_lambda_cli/integrations.py [hunk 2]
```diff
@@ -127,30 +127,28 @@ def create_parameters(
...
 
     template_path = os.path.join(
-        os.path.dirname(os.path.abspath(__file__)), "templates", "import-template.yaml",
+        os.path.dirname(os.path.abspath(__file__)), "templates", "import-template.yaml"
     )
     with open(template_path) as template:
-        change_set_name = f"{INGEST_STACK_NAME}-IMPORT"
-        click.echo(f"Creating change set {change_set_name}")
+        change_set_name = "%s-IMPORT" % INGEST_STACK_NAME
+        click.echo("Creating change set: %s" % change_set_name)
```
Comment (MattWhelan) [inline newrelic_lambda_cli/integrations.py:151]: Why the change to old-style string formatting? Is there something I don't know about Python interpolation?

Comment (kolanos) [reply newrelic_lambda_cli/integrations.py:151]: Yes, f-strings are a 3.6+ feature. The CLI currently supports 3.3+.
